src/pages/auth/VuesticLogo.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <svg
+    :width="112 * scale + 'px'"
+    :height="12 * scale + 'px'"
+    viewBox="0 0 112 12"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M4.96875 12L0 0.375H2.86875L5.1375 5.79375L7.40625 0.375H10.275L5.30625 12H4.96875Z"
+      :fill="colorComputed"
+    />
+    <path
+      d="M16.9875 7.8375V0.375H19.725V7.425C19.725 8.925 20.4375 9.45 21.3938 9.45C22.4438 9.45 23.1 8.925 23.1 7.425V0.375H25.8563V7.8375C25.8563 10.5 23.8688 12 21.3938 12C18.825 12 16.9875 10.425 16.9875 7.8375Z"
+      :fill="colorComputed"
+    />
+    <path
+      d="M33.2812 11.625V0.375H40.725V2.8125H36.0375V4.8375H40.3688V7.2375H36.0375V9.2625H40.9313V11.625H33.2812Z"
+      :fill="colorComputed"
+    />
+    <path
+      d="M47.8313 8.325H50.5125C50.5125 8.98125 51.0375 9.50625 51.675 9.50625C52.3688 9.50625 52.8 9.1125 52.8 8.53125C52.8 7.6875 51.6938 7.44375 50.775 7.125C48.8813 6.45 47.8313 5.56875 47.8313 3.6375C47.8313 1.65 49.5375 0 51.6563 0C54.15 0 55.3313 1.59375 55.4813 3.675H52.8938C52.8938 3.0375 52.5 2.5125 51.7313 2.5125C51.1313 2.5125 50.55 2.86875 50.55 3.6C50.55 4.44375 51.5813 4.59375 52.5375 4.875C54.5438 5.475 55.4813 6.6 55.4813 8.3625C55.4813 10.35 53.7938 12 51.675 12C49.2188 12 47.8313 10.35 47.8313 8.325Z"
+      :fill="colorComputed"
+    />
+    <path d="M64.7625 11.625V2.8125H62.25V0.375H70.0875V2.8125H67.5375V11.625H64.7625Z" :fill="colorComputed" />
+    <path d="M76.9875 11.625V0.375H79.7438V11.625H76.9875Z" :fill="colorComputed" />
+    <path
+      d="M87.2812 6C87.2812 2.64375 89.8125 0 93.375 0C95.5125 0 96.9938 0.80625 98.1938 2.0625L96.3 3.8625C95.5125 3.09375 94.5 2.60625 93.375 2.60625C91.3688 2.60625 90.075 4.10625 90.075 6C90.075 7.89375 91.3688 9.39375 93.375 9.39375C94.5 9.39375 95.5125 8.90625 96.3 8.1375L98.1375 9.9375C97.0125 11.1188 95.4562 12 93.375 12C89.8125 12 87.2812 9.35625 87.2812 6Z"
+      :fill="colorComputed"
+    />
+    <path
+      d="M106.553 8.58593C105.84 8.58593 105.308 8.3802 104.959 7.96874C104.616 7.55208 104.444 6.90885 104.444 6.03906V3.41406H105.788V6.19531C105.788 6.64843 105.847 6.96874 105.967 7.15624C106.092 7.33854 106.288 7.42968 106.553 7.42968C106.819 7.42968 107.014 7.33854 107.139 7.15624C107.269 6.96874 107.334 6.64843 107.334 6.19531V3.41406H108.631V6.03906C108.631 6.90885 108.462 7.55208 108.123 7.96874C107.785 8.3802 107.261 8.58593 106.553 8.58593Z"
```

Comment:
Must be Vuestic Admin, not Vuestic UI mb?

Reply:
Nice catch!